### ISSUE TITLE
Ask for help when loaded the train model

### ISSUE THREAD

**Issue by silencecmsj** [katerynaCh/multimodal-emotion-recognition#3]
Hi, first, thank you for your work. I found the following problems when I loaded your train model. I guess that the train model  provided did not match the original code. How to solve this problem？
![F6 RI44N4{X7TE_WA` BN6G](https://user-images.githubusercontent.com/39476121/206914246-f0f61034-cb4b-4c88-9bc9-dc45e399ef8c.png)

**Comment by katerynaCh** [comment]
Hi, could you specify which model you used?

**Comment by She-yh** [comment]
> Hi, could you specify which model you used?

@katerynaCh 
Hi, I met the same Error.
I tried to use `model.load_state_dict() ` to load the trained model you provided in README `lt_1head_moddrop_2.pth`.
And the Error occurred.

Thank you for your share,
Best regards

**Comment by katerynaCh** [comment]
@silencecmsj @She-yh Are you specifying the correct fusion method when running the script? Note that first two letters of the pretrained model filename correspond to the fusion type which you need to specify as an argument. In this case, you need to specify --fusion lt (correspondingly, --fusion it, or --fusion ia for other models)

**Comment by She-yh** [comment]
@katerynaCh 
Hi, thank you for your reply.
I've made sure that the fusion option fit the model's filename.
This is how I load the model.
```
opt = parse_opts()
model, parameters = generate_model(opt)
state = torch.load('models/lt_1head_moddrop_2.pth')
model.load_state_dict(state['state_dict'])
```
And this is how I run my code.
`
python myscript.py --fusion lt
`
Did I miss other options?

However, If I set strict mode to False like this `model.load_state_dict(state['state_dict'],False)`, the Error disappear. But, I don't know whether this will affect the outcome?

Thank you again for your work, this is of great help to me!
Best regards!
